.swiftlint.yml

```diff
@@ -5,3 +5,9 @@ disabled_rules:
 - line_length
 - type_body_length
 - file_length
+custom_rules:
+  override_func: # rule identifier
+    name: "override in func" # rule name. optional.
+    regex: "(override open) | (override public)" # matching pattern
```

Comment:
Maybe a regex like `override (open|public|private|internal|fileprivate)` would be better to pick up all access control keywords and to prevent duplication of override?

Reply:
Thanks for the suggestion @Sherlouk. I have made required changes again.